training/src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package wolox.training.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import wolox.training.models.Book;
+
+public interface BookRepository extends JpaRepository<Book, Long> {
+
+    Book findByAuthor(String author);
```

Comment:
What happens if there is more than one book with the same author? (It's a real question, I want you to try it)